_sass/homepage/_text-cards.scss

```diff
@@ -0,0 +1,84 @@
+.textcards-section {
+  flex-direction: column;
+  align-items: center;
+
+  @media screen and (max-width: 768px) {
```

Comment:
We should be able to switch to the breakpoint-aware padding and margin utilities now with #314 merged.

Reply:
Swapped in https://github.com/isomerpages/isomerpages-template/pull/313/commits/706b3472e15e47673c22c6300af3e073f5050449!